opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -0,0 +1,436 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=no-member
+
+import abc
+import atexit
+import concurrent.futures
+import logging
+import os
+import threading
+from typing import Any, Callable, Optional, Tuple, Union, cast
+
+from opentelemetry.sdk.environment_variables import (
+    OTEL_PYTHON_LOG_EMITTER_PROVIDER,
+)
+from opentelemetry.sdk.logs.severity import (
+    SeverityNumber,
+    std_to_opentelemetry,
+)
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.util.instrumentation import InstrumentationInfo
+from opentelemetry.trace import get_current_span
+from opentelemetry.trace.span import TraceFlags
+from opentelemetry.util._providers import _load_provider
+from opentelemetry.util._time import _time_ns
+from opentelemetry.util.types import Attributes
+
+_logger = logging.getLogger(__name__)
+
+
+class OTELLogRecord:
```

Comment:
Is this prefixed with `OTEL` to avoid confusion with `logging.LogRecord`? Nothing else in this package has the OTEL prefix so this feels a bit inconsistent. May be we can live with `LogRecord` and let users figure out which one they're using. Somewhat similar to api/sdk objects with same name. 

Admittedly, `LogRecord.from_log_record(log_record)` does feel a bit weird though but perhaps it won't be as bad if users import as aliases like we do with `trace_api`. We could also rename the method to something like `from_std_log_record(record: logging.LogRecord)` to make things a bit more obvious.

Reply:
Yes, it was an attempt to avoid the confusion. On a second thought, I agree letting users to figure it out is fine.